src/main/java/org/jabref/gui/entryeditor/CommentsTab.java

```diff
@@ -151,6 +151,19 @@ protected void setupPanel(BibEntry entry, boolean compressed) {
             });
             gridPane.add(hideDefaultOwnerCommentButton, 1, gridPane.getRowCount(), 2, 1);
             setCompressedRowLayout();
+        } else {
+            Button showDefaultOwnerCommentButton = new Button(Localization.lang("Show user comments"));
```

Comment:
This code is somehow similar to the if branch above. Can you factor out a method with parameters (e.g., the label of the button and whether the usercommentsfield should be shown)